include/z64player.h

```diff
@@ -557,15 +567,11 @@ typedef struct Player {
     /* 0x06A8 */ Actor*     unk_6A8;
     /* 0x06AC */ s8         unk_6AC;
     /* 0x06AD */ u8         unk_6AD;
-    /* 0x06AE */ u16        unk_6AE;
-    /* 0x06B0 */ s16        unk_6B0;
+    /* 0x06AE */ u16        rotsNoRestFlags; // See `NOREST_ROT_` macros. If its flag isn't set, a rot steps to a rest value.
```

Comment:
Why "No Rest"?

Reply:
when the flag is set, the rotation does *not* step to a rest value

Reply:
I see. Next question, what is a rest value? Something like the idle value?

Reply:
That's a good question and I'm open to suggestions on changing that

I got the term from Blender (a 3d modeling piece of software https://www.blender.org/ ), let me define a few terms

- armature = skeleton, made of bones
- pose = rotations and locations of the bones in an armature

For example when animated the armature has a different pose every frame

The "rest pose" is the pose the armature has when all its bones aren't transformed (all rotations/translations/... are 0/identity)

-------

Back to OoT, all rotations involved with `NOREST_ROT_` are stepped to 0 when their flag isn't set
For all these rotations, 0 corresponds visually to a neutral pose of Link, with only the animation affecting his bones, so I refered to those as "rest" values

Also the use case for these flags is when setting these rots to specific values, typically to look at something, which also can be seen as deviating from the "rest" animation-provided pose

Reply:
Could you maybe think of this as a "hold and release" system? So the `rot` is held to a target when the flag is set, and released back to rest when the flag is unset? So something like `HOLD_ROT_`? Or would that conflate with the "HOLD" action of "holding" something above Link's Head?

Reply:
For example in the else block in `func_80836AB8`, the rotations aren't simply set but stepped to different values, so "hold" may be confusing in the sense the values can be still moving, just not to 0/rest, but it may not be such a big deal?

Reply:
It's a subtle difference, but what about `nonRestFlags`? For some reason, sounds better to me but can't articulate why. Might just be me

Reply:
Maybe it's because `no rest` is two words, so it feels more separate, while `non-rest` is one word, so it's easier to group together as a single concept?

Reply:
I need to review the pr in full again, but I think in general itd be good to try a positive name instead of a negative one, like "modified" instead of "no rest" (which i interpret to mean "non default, offset from the rotation the animation is applying)

Reply:
It is related to three rotations / rotation offsets: focus, head, upper (body)

I'm not sure what's up with the focus one, but head and upper are I guess used to make Link face something

The flags control if these rotations are stepped toward "0" (not actually always 0 but something that is "normal"/"default" hence "rest")
- flag unset = rotations are stepped toward "0"
- flag set = rotations are not stepped toward "0"

Phrased this way it makes it arduous to make a positive name but I'm not sure how to phrase it another way 

maybe like "flag set = rot is active" but idk I don't have many ideas

Reply:
From what I can tell down below, it seems like setting this flag just stops rotation back to the rest position (0). You might have been slightly joking, but `activeRot` doesn't seem too bad. Could possibly also do something like `stayRot/remainRot`, or maybe `keepRot`?

Reply:
Just brainstorming: what about something a little more general like `overrideRot` or `overrideRotFlags`, and leave it to the comments to describe that the default behaviour is to restore the rot to resting position, and that by setting this flag, it gives you manual control to control the rotations?